emboss.md

```diff
@@ -0,0 +1,21 @@
+# Support Importing Emboss Files for Serialization/Deserialization Functions
+This project is an example of how to import emboss files(emb file), convert the file into essential yaml file and generate cpp code with modified template.
+
+## Setup
```

Comment:
Add an additional section introducing the reader to Emboss.

Add a link to https://github.com/google/emboss/